merger/includes/veteran-navigation.html

```diff
@@ -0,0 +1,187 @@
+                  <li><a href="/education/" onClick="reportHeaderNav('Explore Benefits->Education Training');">Education and Training Benefits Overview</a></li>
+                  <li><a href="/education/eligibility/" onClick="reportHeaderNav('Explore Benefits->Education Training');">Eligibility</a></li>
+                  <li><a href="/education/apply/" onClick="reportHeaderNav('Explore Benefits->Education Training');">How to Apply for Education Benefits</a></li>
+                  <li><a href="/education/gi-bill/" onClick="reportHeaderNav('Explore Benefits->Education Training');">GI Bill Programs</a></li>
+                  <li><a href="/employment/vocational-rehab-and-employment" onClick="reportHeaderNav('Explore Benefits->Vocational Rehab and Employment');">Vocational Rehabilitation &amp; <br>Employment (VR&amp;E)</a></li>
+                  <li><a href="/gi-bill-comparison-tool/" onClick="reportHeaderNav('Explore Benefits->Education Training');">Compare GI Bill Benefits</a></li>
+                  <li><a href="/education/apply/" class="usa-button va-button-primary" onClick="reportHeaderNav('Explore Benefits->Education Training');">Apply Now</a></li>
+                </ul>
+>>>>>>> master
+              </li>
+              <li>
+                <a href="https://www.va.gov/health/appointments.asp">Schedule a VA Appointment</a>
+              </li>
+              <li>
+                <a href="https://www.veteranscrisisline.net">Crisis Prevention</a>
+              </li>
+              <li>
+                <a href="http://www.mentalhealth.va.gov/">Mental Health</a>
+              </li>
+              <li>
+                <a href="http://www.ptsd.va.gov/">PTSD</a>
+              </li>
+              <li>
+                <a href="http://www.publichealth.va.gov/">Public Health</a>
+              </li>
+              <li>
+                <a href="http://www.va.gov/opa/choiceact/">Veterans Access, Choice and Accountability Act </a>
+              </li>
+            </ul>
+          </div>
+        </li>
+
+        <!-- ABOUT VA -->
+        <!-- first menu item; triggers submenu -->
+        <li>
+          <button class="vetnav-level1 vetnav-trigger" role="button" aria-haspopup="true" aria-controls="vetnav-about-va" onClick="window.dataLayer.push({ event: 'nav-header-top-level' });">About VA</button>
+          <!-- menu container -->
+          <div id="vetnav-about" class="vetnav-panel" role="none" hidden>
+            <ul role="menu" aria-label="About VA">
+              <li>
+                <button class="vetnav-level2 vetnav-trigger" role="button" aria-haspopup="true" aria-controls="vetnav-inside-va" onClick="window.dataLayer.push({ event: 'nav-header-second-level' });">Inside VA</button>
+
+                <!-- submenu panel -->
+                <ul class="vetnav-panel vetnav-panel--submenu" id="vetnav-about-va" role="menu" aria-label="Inside VA" hidden>
+                  <!-- submenu trigger -->
+                  <li>
+<<<<<<< HEAD
```

Comment:
Some merge conflicts here—can you do a global search on the branch for conflict tags like this to make sure these don't get inadvertently merged?

Reply:
Thanks, I didn't see this.